docs/user_guide.md

```diff
@@ -1,15 +1,17 @@
 ## Quick Start
-1. Check [dependencies](#dependencies)
-2. Fill the CSV [config file](#csv-config-file)
-3. Generate the [dashboard](#how-to-generate-dashboard)
-4. Profit!
+1. Ensure that you have the necessary [dependencies](#dependencies).
+2. Read up on [How to Generate Dashboard](#how-to-generate-dashboard).
+3. Fill up the CSV [configuration file](#csv-config-file).
+4. Perform the execution to generate the [dashboard](#dashboard).
+
 
 ## Dependencies
-1. **JDK `1.8.0_60`**  or later
-2. **Git** on the command line
-3. **findstr** for Windows, **grep** for macOS or Linux on the command line
+1. **JDK `1.8.0_60`**  or later.
+2. **Git** on the command line.
+3. **findstr** for Windows, **grep** for macOS or Linux on the command line.
```

Comment:
Maybe we can give an example to test both the commands as the users might not what they mean and they might not know how to check if these tools are installed